cli/src/pcluster/resources/custom_resources/custom_resources_code/cleanup_resources.py

```diff
@@ -76,35 +76,23 @@ def _delete_s3_artifacts(event):
     It exits gracefully if directory does not exist.
     :param bucket_name: bucket containing cluster artifacts
     :param artifact_directory: directory containing artifacts to delete
-    :param remove_bucket: whether or not to remove the bucket, remove only if remove_bucket == "True"
     """
     bucket_name = event["ResourceProperties"]["ResourcesS3Bucket"]
     artifact_directory = event["ResourceProperties"]["ArtifactS3RootDirectory"]
-    remove_bucket = event["ResourceProperties"]["RemoveBucketOnDeletion"]
     try:
         if bucket_name != "NONE":
             bucket = boto3.resource("s3", config=boto3_config).Bucket(bucket_name)
-            if remove_bucket == "True":
-                logger.info("S3 bucket %s deletion: STARTED", bucket_name)
-                bucket.objects.all().delete()
-                bucket.object_versions.delete()
-                bucket.delete()
-                logger.info("S3 bucket %s deletion: COMPLETED", bucket_name)
-            else:
-                logger.info("Cluster S3 artifact under %s/%s deletion: STARTED", bucket_name, artifact_directory)
-                bucket.objects.filter(Prefix="%s/" % artifact_directory).delete()
-                bucket.object_versions.filter(Prefix="%s/" % artifact_directory).delete()
-                logger.info("Cluster S3 artifact under %s/%s deletion: COMPLETED", bucket_name, artifact_directory)
+            logger.info("Cluster S3 artifact under %s/%s deletion: STARTED", bucket_name, artifact_directory)
+            bucket.objects.filter(Prefix="%s/" % artifact_directory).delete()
```

Comment:
nit: let's be consistent in string formatting and use f-strings everywhere?

Reply:
done